### DISCUSSION TITLE
[Feature]: Show Solutions

### DISCUSSION THREAD

**Discussion by rappen** [Power-Maverick/DataverseDevTools-VSCode#11] [Ideas]
### Is your feature request related to a problem?

No

### Describe the problem?

It will be easier to work in a Solution (and by a "must") and limit all gazillian entities.

### Provide a solution or an idea you would like to see in the tool?

So keep the structure:

- Dev
  - Orgs
    - Solutions
      - Tables
        - Columns

or split into:
- Connections
  - Dev
    - Orgs
      - Solutions
and:
- Entities
  - Columns

or perhaps:
- Connections
  - Dev
    - Orgs
and:
- Solutions
  - Entities
    - Columns

For working "bad" without solution, it can have a "solution" for "Default Solution", showing e v e r y t h i n g.

[a comment by an automated account is not shown]

**Comment by Power-Maverick** [comment]
Yes @rappen that is one of the planned feature. 

https://github.com/Power-Maverick/DataverseDevTools-VSCode#-planned-features

**Comment by Power-Maverick** [comment]
I have implemented it differently (see [here](https://github.com/Power-Maverick/DataverseDevTools-VSCode#filter-by-solution)). Do you think it is beneficial to have solutions as a tree view as well?

**Comment by PauloBiai** [comment]
I'm trying to implement data analysis with python and i need to connect to my tables on CSD, how do i do that?

 

Please share any thoughts/suggestions on this. Thank you.

**Reply by Power-Maverick** [reply]
That is currently not supported.
